### PR TITLE
Adds Bar Chart Docs

### PR DESCRIPTION
**Pull request by lukeschunk**
Fixes: https://github.com/mxenabled/mx-react-components/issues/446 

I realized there were no docs for the BarChart component, so I added these docs. 🔮

### CODE REVIEW

File: docs/components/BarChartDocs.js [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+  },
+
+  render () {
+    const styles = this.styles();
+
+    return (
+      <div>
+        <h1>
+          BarChart
+          <label>CHANGE THIS A D3 donut chart that supports a center label, multiple slices, and reference dots.</label>
```
Comment (jtanner) [inline docs/components/BarChartDocs.js]: I think you missed this line.

Comment (lukeschunk) [inline docs/components/BarChartDocs.js]: 👍

Comment (lukeschunk) [inline docs/components/BarChartDocs.js]: @jtanner updated the description but take a look and see if you would change it - i'm not  a huge fan of what I wrote

File: docs/components/BarChartDocs.js [hunk 2]
```diff
@@ -96,7 +96,7 @@ const BarChartDocs = React.createClass({
       <div>
         <h1>
           BarChart
-          <label>CHANGE THIS A D3 donut chart that supports a center label, multiple slices, and reference dots.</label>
+          <label>CHANGE THIS A D3 bar chart that supports labels for the bars and tooltips.</label>
```
Comment (jtanner) [inline docs/components/BarChartDocs.js]: Should `CHANGE THIS ` be removed?

Comment (jtanner) [inline docs/components/BarChartDocs.js]: 👍

**Comment by lukeschunk** [comment]
removed change this @jtanner

**Comment by jtanner** [comment]
:shipit:

**Pull request merged**